src/components/analyses/listing/Listing.js

```diff
@@ -650,9 +648,14 @@ function Listing(props) {
 
     const getSelectedAnalyses = (analyses) => {
         const items = analyses ? analyses : selected;
-        return items.map((id) =>
-            data?.analyses.find((analysis) => analysis.id === id)
-        );
+        if (items) {
+            return items
+                .map((id) =>
+                    data?.analyses.find((analysis) => analysis.id === id)
+                )
+                .filter((item) => item !== null && item !== undefined);
+        }
+        return null;
```

Comment:
```suggestion
        return items?.map((id) =>
                data?.analyses.find((analysis) => analysis.id === id)
            )
            .filter((item) => item !== null && item !== undefined);
```
Also, not saying we shouldn't have the `filter` check, but seems like we should fix what's causing null values to get added to either `analyses` or `selected`.

Reply:
This happening due to a race condition. When HT Drill down icon is clicked, that row is `selected` but in the background we start fetching the jobs associated with HT job. At this point the state is cleared and the `find` method can no longer find the analysis matching the selection and I think that results in `find` returning an array `[null]`. I don't know how to fix this yet.